hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -25,7 +25,6 @@
 
   // Add additional tops for simulation.
   sim_tops: ["clkmgr_bind",
-             "pwrmgr_bind",
```

Comment:
Please don't remove pwrmgr_bind: there are more assertions than pwrmgr_ast, so you are removing more checks. Instead you could rename dv/sva/pwrmgr_rstmgr_bind.sv to something like pwrmgr_unit_bind.sv and place the version of pwrmgr_ast_sva_if suitable for unit testing (with no clocks) there. That way we can continue to bind pwrmgr_bind in top-level with the other assertions.

Then I would put the pwrmgr_ast_sva_if suitable for top-level in top_earlgrey_bind. This will make it easier to understand the flow, since it would be more like the way we handle pwrmgr_rstmgr_sva_if which has a similar discrepancy between top and unit level.